lib/Runtime/Library/IntlEngineInterfaceExtensionObject.cpp

```diff
@@ -566,7 +566,7 @@ namespace Js
         }
 
 #if DEBUG
-#ifdef INTL_ICU_DEBUG
+#if INTL_ICU_DEBUG
```

Comment:
Side note: FWIW I'm not a fan of this change. I commented on your other PR as well about the idea here, but the fact that there is a value given to this macro in `#define` is defensive -- the point is actually that it's defined at all.

Reply:
The reason for the change was to allow `if (INTL_ICU_DEBUG) Output::Print()` in the ICU_ASSERT/ICU_RETURN macros, since _as far as I know_ (I may be very wrong), I cant have #ifdefs in macros (I would need to conditionally define the entire macro just for printing purposes). Though, thinking about it, I suppose I could just macro the Output::Print part conditionally on its own?